### PR TITLE
Make five iterations

### PR DESCRIPTION
**Pull request by ZdrGeo**

### CODE REVIEW

File: BoxSand/BoxSand/Program.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ static void Main(string[] args)
         {
             Console.WriteLine("test");
             Console.WriteLine("This is F1 merged to dev");
-            for (int i = 0; i < 2; i++)
+            for (int i = 0; i < 5; i++)
```
Comment (peppial) [inline BoxSand/BoxSand/Program.cs]: Can you move that to a const please.

Comment (ZdrGeo) [inline BoxSand/BoxSand/Program.cs]: Sure!

Comment (peppial) [inline BoxSand/BoxSand/Program.cs]: test

Comment (ZdrGeo) [inline BoxSand/BoxSand/Program.cs]: Done

**Pull request merged**